lib/fog/azurerm/requests/network/detach_route_table_from_subnet.rb

```diff
@@ -0,0 +1,53 @@
+module Fog
+  module Network
+    class AzureRM
+      # Real class for Network Request
+      class Real
+        def detach_route_table_from_subnet(resource_group, subnet_name, virtual_network_name, address_prefix, network_security_group_id)
```

Comment:
Actually I explicitly added 'from_sbnet' string so that is clear to the reader that this call is related to subnet. Because we mention all requests in network service file and you can differentiate which request is related to which model.